framework/mesh/surface_mesh/surface_mesh.cc

```diff
@@ -197,7 +198,7 @@ SurfaceMesh::ImportFromOBJFile(const std::string& fileName, bool as_poly, const
   if (not file.is_open())
   {
     std::ostringstream oss;
-    oss << fname << ": Failed to open file: " << fileName;
+    oss << "SurfaceMesh: Failed to open file " << fileName;
```

Comment:
If yo do this, can you check that you are not getting a warning on `fname` that would be "assigned into but never used"

Reply:
Fixed. I removed the unused `fname` variables.